src/lib/items/Item.js

```diff
@@ -55,7 +53,9 @@ export default class Item extends Component {
     groupTops: PropTypes.array,
     useResizeHandle: PropTypes.bool,
     moveResizeValidator: PropTypes.func,
-    onItemDoubleClick: PropTypes.func
+    onItemDoubleClick: PropTypes.func,
+
+    scrollRef: PropTypes.object

```

Comment:
Yes! this is a far better approach :)

Thank you! I will change to make Timeline.js calculate the offsets if an item is selected. Pass these to <Items /> and then to each <Item />

Reply:
I have chosen not to do this. Currently the parent elements do not listen for vertical scrolls or offset changes.

I think a better method is the current to calculate values of the DOM when the item is being dragged. Otherwise we need to attach listeners, state and pass in as props. 

Or is there a better way?